eZ/Publish/API/Repository/Values/User/RoleCreateStruct.php

```diff
@@ -25,6 +25,13 @@
     public $identifier;
 
     /**
+     * The status of the role.
+     *
+     * @var int One of Role::STATUS_DEFINED|Role::STATUS_DRAFT
+     */
+    public $status = Role::STATUS_DRAFT;
```

Comment:
Why is this needed? AFAICS this makes possible to create defined Role through `RoleService::createRole(RoleCreateStruct $roleCreateStruct)`, which returns RoleDraft.